.circleci/config.yml

```diff
@@ -693,7 +693,7 @@ jobs:
       - run_brew_for_macos_build
       - run:
           name: Test
-          no_output_timeout: "1h"
```

Comment:
Hmm, I think this might bite us later on, but we can think about a fix for this later.  When you pipe everything into the temp file and only print it at the end, the outputs are nicely separated but won't be interactive. I guess this is the reason why you need to increase this `no_output_timeout` limit.

```
with open(log_path, "w") as f:
    ret_code = shell(command, test_directory, stdout=f, stderr=f, env=env)
print_log_file(test_module, log_path)
```

IMO, a fix would be to have something print out while waiting for the sub-processes, something like:

```
while pool not join:
  print("Waiting ...")
  sleep(5)
```

Reply:
yes thats right, i considered doing a print thing so people know its alive, but im worried it would just create more logs than necessary